sci-physics/cuttools/cuttools-1.8.2.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Computing 1-loop amplitudes at the integrand level"
+HOMEPAGE="https://www.ugr.es/~pittau/CutTools/"
+SRC_URI="http://www.ugr.es/~pittau/CutTools/${PN}_v${PV}.tar.gz"
+S="${WORKDIR}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="+dummy"
+DEPEND="
+	sci-physics/qcdloop
+	sci-physics/oneloop[dpkind,qpkind16,-qpkind,-tlevel]
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	virtual/fortran
+"
```

Comment:
`fortran-2.eclass` :P

Reply:
I am going to patch that now into my pkgcheck...